scopesim/effects/spectral_trace_list_utils.py

```diff
@@ -86,15 +86,14 @@ def fov_grid(self):
         Spatial limits are determined by the `ApertureMask` effect
         and are not returned here.
         """
-        trace_id = self.meta["trace_id"]
         aperture_id = self.meta["aperture_id"]
         lam_arr = self.table[self.meta["wave_colname"]]
 
         wave_max = np.max(lam_arr)
         wave_min = np.min(lam_arr)
 
         return {"wave_min": wave_min, "wave_max": wave_max,
-                "trace_id": trace_id, "aperture_id": aperture_id}
+                "trace_id": self.trace_id, "aperture_id": aperture_id}
```

Comment:
Where is `self.trace_id` first defined? I can't find it in `__init__`

Reply:
Or is this one of those new-fangled things that `_class_param` dict entries can be accessed through `self.`?